packages/manager/apps/hub/src/pages/layout/layout.tsx

```diff
@@ -0,0 +1,211 @@
+import React, { useEffect, useContext, useRef, Suspense, lazy } from 'react';
```

Comment:
The page naming convention is: `PageName.page.tsx` according to the documentation https://manager.pages.ovhcloud.tools/guides/manager/development-guidelines/application-folder-structure/, unless there is an exception for the layout page?

Reply:
There is indeed an exception for layout, but here maybe this is not simply a layout but really a page.
I'll see later if it really seems that way to modify it consequently.